File: app/src/main/java/com/saint/struct/tool/InjectTest.java
package com.saint.struct.tool;

import android.util.Log;

import com.saint.annotation.PluginTest;


public class InjectTest {
    @PluginTest
    public void sayHello(){
        System.out.println("你好");
        try {
            Thread.sleep(1000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/com/saint/struct/tool/InjectTest.kt
package com.saint.struct.tool

import com.saint.annotation.PluginTest

class InjectTest {
    @PluginTest
    fun sayHello() {
        println("你好")
        try {
            Thread.sleep(1000)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
    }
}